src/qt/overviewpage.cpp

```diff
@@ -124,8 +124,8 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
 {
     ui->setupUi(this);
 
-    // use a SingleColorIcon for the "out of sync warning" icon
-    QIcon icon = platformStyle->SingleColorIcon(":/icons/warning");
+    // use red icon for the "out of sync warning" icon
+    QIcon icon(":/icons/outofsync");
```

Comment:
Instead of duplicating the icon, what about `ColorizeImage()` from platformstyle.cpp?